README.md

```diff
@@ -0,0 +1 @@
+# araig_test_stack
```

Comment:
Please Insert:

# ARAIG Test Stack
This stack contains the GUI, calculators, interpreters and runners required to implement any ARAIG test. 

The actual launch files per robot will be stored in [araig_robot_packages](https://github.com/ipa320/araig_robot_packages)

Benchmarking for these components per test will be stored in [araig_benchmarks](https://github.com/ipa320/araig_benchmarks)

## Requirements

Install python dependencies not supported by rosdep with:
`pip install -m requirement.txt`